.gitlab/test_plugins.sh

```diff
@@ -45,4 +48,4 @@ uname -a || true
 
 echo "==== Running Plugins Gtest tests ===="
 cd ${INSTALL_DIR}
-./bin/plugins_gtest
+./bin/plugins_gtest --min-tcp-port="$(min_gtest_port)" --max-tcp-port="$(max_gtest_port)"
```

Comment:
need to use ` --min-tcp-port="$min_gtest_port" --max-tcp-port="$max_gtest_port"`